pycbc/events/stat.py

```diff
@@ -336,6 +339,20 @@ def get_hist(self, ifos=None):
                 self.param_bin[ifo] = param
                 self.pdtype = self.param_bin[ifo].dtype
             self.max_penalty = self.weights[ifo].min()
+            if self.two_det_flag:
+                # 3bins, for "t", "p", and "s". However, we can expand the
+                # weights lookup table here, avoiding the need to not store 0
+                # values by using param_bin. This makes the lookup a O(N)
+                # rather than O(NlogN) operation. It sacrifices RAM to do this,
+                # so is a good tradeoff for 2 detectors, but not for 3!
+                array_size = [256,256,256]
```

Comment:
Didn't @GarethDaviesGW propose a way to avoid hardcoding the size?

Reply:
Sorry, we crossed wires here. I was just implementing this change while addressing the CC comments. Please see now.